any_parser/any_parser.py

```diff
@@ -20,6 +21,10 @@
 from any_parser.utils import validate_file_inputs
 
 PUBLIC_SHARED_BASE_URL = "https://public-api.cambio-ai.com"
+# TODO: Update this to the correct batch endpoint
+PUBLIC_BATCH_BASE_URL = (
+    "http://AnyPar-ApiCo-cuKOBXasmUF1-1986145995.us-west-2.elb.amazonaws.com"
```

Comment:
let me link this to our DNS.

Reply:
<img width="906" alt="image" src="https://github.com/user-attachments/assets/0d0fb589-d8bc-4e1a-819c-dc26365a7c27">
I just created batch-api domain name forward for batch-api.cambio-ai.com. However, one thing I am not 100% sure is that for API gateway, we used to setup custom domain names in it. This is a directly public elb, so do we have to do anything on the AWS side besides the squarespace domains work that I have already done. I can merge this in first and you can address in a future PR.